### PR TITLE
Upgrade to terraform 0.12.x

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -100,24 +100,30 @@ json/format: | guard/program/jq
...
 	cat $(README_PARTS) > $(README_FILE)
 	@ echo "[$@]: Documentation files creation complete!"
 
-dep/install: guard/program/curl
-	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
-terratest/install: | guard/program/go guard/program/dep
-	cd tests && dep ensure
+terratest/install: | guard/program/go
+	cd tests && go mod init terraform-aws-tardigrade-inspector/tests
```
Comment (lorengordon) [inline Makefile]: inspector => iam-account?

File: tests/go.mod [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module terraform-aws-tardigrade-inspector/tests
```
Comment (lorengordon) [inline tests/go.mod]: inspector => iam-account?

File: Makefile (hunk 1 above)
Comment (confusdcodr) [reply Makefile]: Ah, yes. Sorry copy/pasta error

Comment (confusdcodr) [reply Makefile]: Fixed both spots

**Pull request merged**